modules/imgproc/test/test_color.cpp

```diff
@@ -3072,20 +3072,42 @@ TEST(ImgProc_RGB2YUV, regression_13668)
     EXPECT_EQ(res, ref);
 }
 
-TEST(ImgProc_cvtColorTwoPlane, missing_check_17036)  // test can be removed if required feature is implemented
+TEST(ImgProc_cvtColorTwoPlane, regression_17036)
```

Comment:
Let's rename it according to the test logic.

Reply:
Done.